packages/keyda-demo/src/lib/FormToSubmit/index.js

```diff
@@ -2,39 +2,58 @@ import React, { useCallback } from 'react';
 import _ from 'lodash';
 import axios from 'axios';
 
-import { useKeyStateState } from '../Context';
+import { useKeyStateState, useKeyStateDispatch } from '../Context';
 
 const FormToSubmit = (props) => {
-  const definedFormTypes = ['REGISTER', 'LOGIN', 'PASSWORD_CHANGE'];
-
+  const definedFormTypes = ['REGISTER', 'LOGIN', 'CHANGE_PW'];
   const { onSubmit, formType, children, ...rest } = props;
 
   if (!formType) {
     throw new Error(
-      'You should pass the "formType" prop explicitly to define a mechanism of KeyDa form.\nYou can choose one of "REGISTER", "LOGIN", "PASSWORD_CHANGE" matched with your usage.'
+      'You should pass the "formType" prop explicitly to define a mechanism of KeyDa form.\nYou can choose one of "REGISTER", "LOGIN", "CHANGE_PW" matched with your usage.'
     );
   } else if (!_.includes(definedFormTypes, formType)) {
     throw new Error(
-      'You passed wrong "formType". You can choose one of "REGISTER", "LOGIN", "PASSWORD_CHANGE" matched with your usage.'
+      'You passed wrong "formType". You can choose one of "REGISTER", "LOGIN", "CHANGE_PW" matched with your usage.'
```

Comment:
Changed `formType` value to more usable and readable when we use this for the URL parameter.